v3.9/introduction/index.md

```diff
@@ -43,7 +43,7 @@ custom_css: css/intro.css
   </div>
   <div class="col-md-6">
     <h3 style="margin-top: 5px">Performance</h3>
-    <p>{{site.prodname}} uses the Linux kernel’s built-in highly optimized forwarding and access control capabilities to deliver native Linux networking dataplane performance, typically without requiring any of the encap/decap overheads associated with first generation SDN networks. {{site.prodname}}’s control plane and policy engine has been fine tuned over many years of production use to minimize overall CPU usage and occupancy.</p>
+    <p>{{site.prodname}} uses the Linux kernel’s built-in, highly optimized forwarding and access control capabilities to deliver native Linux networking dataplane performance (note that this is done in {{site.prodname}} *without* requiring any of the encap/decap overheads associated with first generation SDN networks). {{site.prodname}}’s control plane and policy engine has been fine-tuned over several years in production use cases, and is known to be one of the most efficient CNI implementations for kubernetes in terms of CPU usage and occupancy, while under high load.</p>
```

Comment:
I think we should revert this diff as well, as I don't think it fixes any real problem in the existing text.

Reply:
ok, specifically I was referring to the performance tests which were done relative to other CNIs, https://itnext.io/benchmark-results-of-kubernetes-network-plugins-cni-over-10gbit-s-network-36475925a560 .  IMO indicating that the 'fine-tuning' has been verified with actual measurements has some value...  but we can worry about that in a later PR maybe.